### DISCUSSION TITLE
Can I use Shortcodes locally?

### DISCUSSION THREAD

**Discussion by afridimatin** [holger1411/understrap#1361] [General]
Hello, I'm new to understrap and I was wondering if I could use wordpress shortcodes on my html file?

Thank you

**Comment by danemorgan** [comment]
Do you mean using shortcodes already included in WordPress or in a plugin? If so, yes, you just add the shortcode to the content in your editor or in your theme code with a do_shortcode( 'shortcode_slug' ); function call.

If you mean you want to write new shortcodes, you can either make a folder to store your shortcode declarations and include them from functions.php or add it to functions.php directly. But the better way to go here would be to write a small plugin to add the shortcode.

Shortcodes are independent of themes and are plugin territory.

**Reply by afridimatin** [reply]
Thank you!

**Reply by bacoords** [reply]
Please mark the answer from @danemorgan as accepted if you agree!
